be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -43,10 +43,11 @@ Status VOlapTableSink::init(const TDataSink& sink) {
 }
 
 Status VOlapTableSink::prepare(RuntimeState* state) {
+    RETURN_IF_ERROR(OlapTableSink::prepare(state));
     // Prepare the exprs to run.
     RETURN_IF_ERROR(vectorized::VExpr::prepare(_output_vexpr_ctxs, state, _input_row_desc,
                                                _expr_mem_tracker));
-    return OlapTableSink::prepare(state);
+    return Status::OK();
```

Comment:
Why no need to call `OlapTableSink::prepare(state);`?

Reply:
![image](https://user-images.githubusercontent.com/13197424/162651593-5743ebbb-1eaa-465c-9518-253786a684dc.png)
Changed location because `mem_tracker` is initialized in `OlapTableSink::prepare`.
Testing found no effect on logic.